src/backend/functions/users.ts

```diff
@@ -97,7 +102,7 @@ const logUserIn: ApiRouteFunction = async (_params, event) => {
     }
   });
 
-  return buildSuccessResponse(usersTransformer(user));
+  return { ...buildSuccessResponse(usersTransformer(user)), accessToken };
```

Comment:
Is this the best way to be passing it to the front-end? Did you consider doing a header? I think it has implications on our type interfaces, how the front-end retrieves it, how the front end can send it back, and how this needs to work during local development too. Right now I think the local development login skips sending a POST request to the login endpoint and instead just lets the user into the front end application and fills in the user object on the front end

Reply:
Note: this method of trying to send the token to the client does not appear to work